tests/unit/amo/pages/TestCollection.js

```diff
@@ -1019,6 +1021,10 @@ describe(__filename, () => {
       }),
     });
 
+    const dispatch = jest
```

Comment:
This is what I had to do to resolve the problem of the tests running for a very long time and eventually resulting in a JS oom error. I was never able to fully debug it and trace everything through, but it seems like allowing the server redirect to be fired in the test was causing the issue. Perhaps, for some reason, it was causing an infinite loop? We have other tests where we are checking something similar, and they don't seem to have any issues, so this is a bit weird. I tried adding debugging info, but it just stopped being output at some point. It's probably worth doing a bit more investigation into this, but for now this seems like a reasonable workaround.

Reply:
Just in case we should flag that for QA to double-check once the dependency updates go live.

Reply:
Good idea!

Reply:
I added a comment to https://github.com/mozilla/addons-frontend/pull/11652#issue-1294644653